core/driver/sources/src/main/scala/org/virtuslab/ideprobe/reporting/ErrorValidator.scala

```diff
@@ -1,13 +1,15 @@
 package org.virtuslab.ideprobe.reporting
 
+import scala.collection.mutable
+
 import org.virtuslab.ideprobe.config.CheckConfig
 import org.virtuslab.ideprobe.protocol.IdeMessage
 
 object ErrorValidator {
   def apply(config: CheckConfig, errors: Seq[IdeMessage]): Option[Exception] = {
     val filteredErrors = errors
-      .filter(error => config.errors.includeMessages.exists(_.r.findFirstIn(error.content).nonEmpty))
-      .filterNot(error => config.errors.excludeMessages.exists(_.r.findFirstIn(error.content).nonEmpty))
+      .filter(ideMessageMatchesMessagesFromConfig(_, config.errors.includeMessages))
```

Comment:
Pls name the `_` param; I generally recommend using `_` placeholder only when its usage is instantly obvious (like in the previous `exists`)

Reply:
Or alternatively, curry up `ideMessageMatchesMessagesFromConfig` and then use as
```suggestion
      .filter(ideMessageMatchesMessagesFromConfig(config.errors.includeMessages))
```